src/test/java/org/cactoos/http/HtSecureWireTest.java

```diff
@@ -132,7 +134,12 @@ public InputStream stream() throws IOException {
                 new JoinedText(
                     delimiter,
                     "GET / HTTP/1.1",
-                    String.format("Host: %s", this.host),
+                    new UncheckedText(
```

Comment:
@alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `IOException`

Reply:
Fixed